topology/src/main/java/io/camunda/zeebe/topology/api/AddMembersTransformer.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.topology.api;
+
+import io.atomix.cluster.MemberId;
+import io.camunda.zeebe.topology.changes.TopologyChangeCoordinator.TopologyChangeRequest;
+import io.camunda.zeebe.topology.state.ClusterTopology;
+import io.camunda.zeebe.topology.state.TopologyChangeOperation;
+import io.camunda.zeebe.topology.state.TopologyChangeOperation.MemberJoinOperation;
+import io.camunda.zeebe.util.Either;
+import java.util.List;
+import java.util.Set;
+
+public class AddMembersTransformer implements TopologyChangeRequest {
```

Comment:
:upside_down_face: I'd prefer to name them "AddMembersRequest" (same for the other classes). In my head they are **requests** that _can_ generate a list of operations.

Reply:
Ah I guess it's nice to have a new name to distinguish these classes from the ones generated by protobuf :thinking: 

Feel free to ignore then.